app/assets/stylesheets/admin/loans/_print.scss

```diff
@@ -65,4 +67,26 @@
       min-height: inherit;
     }
   }
+
+  .images {
+    .image-list {
+      display: flex;
+      flex-direction: row;
+      align-items: flex-start;
+      flex-wrap: wrap;
+    }
+
+    .img-container {
+      height: 2in;
+      width: 3.5in;
+      margin: .5em;
+      border: 1px solid $gray3;
+    }
+
+    img {
+      max-height: 100%;
+      max-width: 100%;
```

Comment:
This was confusing at first but now I get it! This basically forces the image to grow to take up as much of the container as it can? Cool! Might be nice to put this as a comment...

Reply:
Yes. That's what is happening.